test/nimble_template/addons/test_env_test.exs

```diff
@@ -20,7 +20,10 @@ defmodule NimbleTemplate.Addons.TestEnvTest do
           assert file =~ """
                    defp aliases do
                      [
-                       codebase: [\"deps.unlock --check-unused\", \"format --check-formatted\"],
+                       codebase: [
+                         \"deps.unlock --check-unused\",
+                         \"format --check-formatted\"
```

Comment:
Does this work?
```suggestion
                         "deps.unlock --check-unused",
                         "format --check-formatted"
```

Reply:
Good catch @rosle I have updated it in https://github.com/nimblehq/elixir-templates/pull/144/commits/5df253ce7739fb4b5a5712e1e6f17d0700b20302